accounting/prisma/schema.prisma

```diff
@@ -141,7 +141,12 @@ model Account {
 
   externalAccount Currency? @relation("externalAccount")
 
-  tags AccountTag[] @relation()
+  tags              AccountTag[]        @relation()
+
+  // Each CreditCommonsNode has a "vostro" Account
+  CreditCommonsNode CreditCommonsNode[]
+
+  @@unique([id, tenantId])
```

Comment:
THe account already has id as primary key, so this `@unique` is redundant. Maybe it would be formally more correct to have the primary key as (id, tenantId), but since id's are UUID I think it is just fine to have a global primary key.

Reply:
Ah, I remember now why I added it; prisma told me to :)
```
michiel-2:accounting michiel$ ./node_modules/.bin/prisma migrate dev --name credit_commons_node
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Error: Prisma schema validation - (get-config wasm)
Error code: P1012
error: Error parsing attribute "@relation": The argument `references` must refer to a unique criterion in the related model. Consider adding an `@@unique([id, tenantId])` attribute to the model `Account`.
  -->  schema.prisma:292
   | 
291 |   vostroId String
292 |   vostro   Account @relation(fields: [vostroId, tenantId], references: [id, tenantId])
293 | 
   | 

Validation Error Count: 1
[Context: getConfig]

Prisma CLI Version : 5.13.0